desktop-src/SbsCs/application-manifests.md

```diff
@@ -111,7 +111,7 @@ The **assemblyIdentity** element has the following attributes. It has no subelem
 | **type**                  | Specifies the application or assembly type. The value must be Win32 and all in lower case. Required.                                                                                                                                                                                                                                                                                                                              |
 | **name**                  | Uniquely names the application or assembly. Use the following format for the name: Organization.Division.Name. For example Microsoft.Windows.mysampleApp. Required.                                                                                                                                                                                                                                                               |
 | **language**              | Identifies the language of the application or assembly. Optional. If the application or assembly is language-specific, specify the DHTML language code. In the **assemblyIdentity** of an application intended for worldwide use (language neutral) omit the language attribute.<br/> In an **assemblyIdentity** of an assembly intended for worldwide use (language neutral) set the value of language to "\*".<br/> |
-| **processorArchitecture** | Specifies the processor. The valid values are x86 for 32-bit Windows and ia64 for 64-bit Windows. Optional.                                                                                                                                                                                                                                                                                                                       |
+| **processorArchitecture** | Specifies the processor. The valid values are `x86`, `amd64`, `arm` and `arm64`. Optional.                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Maybe we should say

> Valid values include `x86`, `amd64`, `arm`, and `arm64`.

That way, this documentation doesn't become instantly *wrong* every time we onboard a new architecture. (It just becomes instantly *suboptimal*, which is better than being instantly wrong.)